pkgs/development/libraries/genomicsqlite/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, cmake, sqlite, zstd, htslib, sqlitecpp }:
+
+let
+  sqlite_zstd_vfs = fetchFromGitHub {
+    owner = "mlin";
+    repo = "sqlite_zstd_vfs";
+    rev = "dadf93e";
+    sha256 = "1aj2g52dy5j9wbmb2flcms36bsb1z3rjvva8f9dfdr5yg1hfvcx0";
+  };
+
+in stdenv.mkDerivation rec {
+  pname = "GenomicSQLite";
+  version = "0.4.0";
```

Comment:
You can update to 0.8.0 now.

Reply:
I've updated to v0.8.0, but things are more complicated now. There's an additional dependency on sqlite_web_vfs (unknown which specific revision, if any, is required), which itself has an undocumented dependency on readerwriterqueue. Readerwriterqueue at least could be a separate package. I'll try contacting the author to see if zstd (and web) can be built separately.